articles/architecture-scenarios/application/mobile-api.md

```diff
@@ -4,35 +4,290 @@ title: Mobile + API
+
+### Create the Client
+
+There are four client types in Auth0: __Native__ (used by mobile or desktop apps), __Single Page Web Applications__, __Regular Web Applications__ and __Non Interactive Clients__ (used by CLIs, Daemons, or services running on your backend). For this scenario we want to create a new Client for our Mobile Application, hence we will use Native as the client type.
+
+To create a new Client, navigate to the [dashboard](${manage_url}) and click on the [Clients](${manage_url}/#/clients}) menu option on the left. Click the __+ Create Client__ button.
+
+Set a name for your Client (we will use `Timesheets Mobile`) and select `Native` as the type.
+
+Click __Create__.
+
+![Create Client](/media/articles/architecture-scenarios/mobile-api/create-client.png)
+
+That's it for now. When we are done with the Mobile app implementation we will revisit the dashboard and this Client's settings to make some changes in its configuration.
+
+## Inside the Implementation
+
+<%= include('../_includes/_api-implement.md') %>
+
+### Implement the Mobile App
+
+In this section we will see how we can implement a Mobile application for our scenario.
+
+::: note
+[See the implementation in Android.](/architecture-scenarios/application/mobile-api/mobile-implementation-android#1-set-up-the-application)
+:::
+
+#### Authorize the User
+
+To authorize the user we will implement an [Authorization Code Grant Flow with PKCE](/api-auth/tutorials/authorization-code-grant-pkce). The mobile application should first send the user to the [authorization URL](/api/authentication#authorization-code-grant-pkce-) along with the `code_challenge` and the method used to generate it:
+
+```text
+https://${account.namespace}/authorize?
+    audience=API_AUDIENCE&
+    scope=SCOPE&
+    response_type=code&
+    client_id=YOUR_CLIENT_ID&
+    code_challenge=CODE_CHALLENGE&
+    code_challenge_method=S256&
+    redirect_uri=https://YOUR_APP/callback
+```
+
+The `GET` request to the authorization URL should include the following values:
+
+Parameter | Description
+----------|------------
+__domain__ | The value of your Auth0 Domain. You can retrieve it from the Settings of your Client at the [Auth0 Dashboard](${manage_url}/#/clients).
+__client_id__ | The value of your Auth0 Client Id. You can retrieve it from the Settings of your Client at the [Auth0 Dashboard](${manage_url}/#/clients).
+__audience__ | The value of your API Identifier. You can retrieve it from the Settings of your Client at the [Auth0 Dashboard](${manage_url}/#/clients).
+__scope__ | The [scopes](/scopes) which determine the information to be returned in the `id_token` and `access_token`. A scope of `openid profile email offline_access` will return all the user profile information the `id_token` and return a `refresh_token`. You also need to request the scopes required to call the API, in this case the `read:timesheets create:timesheets` scopes. This will ensure that the `access_token` has these scopes.
```

Comment:
`A scope of `openid profile email offline_access` will return all the user profile information IN the `id_token`:

"all the user information" is kind of vague. Tell them they need to specify `openid` in order to get an id_token. And add a more complete example, like the one your mobile apps expects, for example if your mobile app is to extract the `name` claim from the token, then tell them to use `openid profile`

See https://auth0.com/docs/scopes/current#example-asking-for-standard-claims

Reply:
I just saw that here http://localhost:3000/docs/architecture-scenarios/application/mobile-api/mobile-implementation-android#create-the-login-activity you are using this: `scope: create:timesheets read:timesheets openid profile email offline_access`

You can say that in our example we use this AND it will return this and that info